tests/test_edgeql_insert.py

```diff
@@ -3643,3 +3643,15 @@ async def test_edgeql_insert_and_delete_01(self):
                     (INSERT Person { name := 'foo' })
                 )
             ''')
+
+    async def test_edgeql_insert_cardinality_assertion(self):
+        async with self.assertRaisesRegexTx(
```

Comment:
Can you add a test for UPDATE, simple & straightforward like this one for INSERT?

Reply:
I already did, see the diff

Reply:
Got it, there was a test already, GH just collapsed the view of it.